backend/capellacollab/sessions/operators/__init__.py

```diff
@@ -4,8 +4,15 @@
 
 from capellacollab.sessions.operators.k8s import KubernetesOperator
 
-OPERATOR = KubernetesOperator()
+
+class OperatorLoader:
+    operator = None
+
+    @classmethod
```

Comment:
Why the class? Just use a module variable. One class with a class method is not pythonic IMHO.

Reply:
Fixed.